lib/utils/theme/theme.dart

```diff
@@ -138,5 +132,10 @@ ThemeData theme(BuildContext context) {
     iconTheme: IconThemeData(
       color: surfaceForegroundColor,
     ),
+    colorScheme: ColorScheme.fromSeed(
+      brightness: brightness,
+      seedColor: accentColor,
+      secondary: accentColor,
+    ).copyWith(background: backgroundColor),
```

Comment:
why not set background color directly from the constructor?

Reply:
The fixes introduced in this PR were purely just hitting CTRL + S and letting the extension fix them automatically so I wouldn't know the answer to your question, this was just replacing deprecated items, not diving deep into theme itself and doing refactors there.

Reply:
oghe then move the thing to the constructor